src/components/form-attachment/row.vue

```diff
@@ -13,15 +13,28 @@ except according to the terms contained in the LICENSE file.
   <tr :class="htmlClass">
     <td class="form-attachment-list-type">{{ type }}</td>
     <td :title="attachment.name" class="form-attachment-list-name">
-      <a v-if="attachment.exists" :href="href" target="_blank">
+      <a v-if="attachment.blobExists" :href="href" target="_blank">
         {{ attachment.name }}
       </a>
       <template v-else>
         {{ attachment.name }}
       </template>
     </td>
     <td class="form-attachment-list-uploaded">
-      <template v-if="attachment.exists">
+      <template v-if="attachment.datasetExists">
+        <span class="icon-link"></span>
+        <span class="dataset-label">
+          <i18n-t keypath="linkedToDataset">
```

Comment:
By default, `<i18n-t>` doesn't render an element, but it's possible to make it do so, and that's something we do a lot. You'd effectively be specifying the `<span>` and the `<i18n-t>` together. Feel free to take that approach if it feels more ergonomic!

```vue
<i18n-t tag="span" keypath="linkedToDataset" class="dataset-label">
```